generated/server/mock_controller.go

```diff
@@ -0,0 +1,64 @@
+// Automatically generated by MockGen. DO NOT EDIT!
+// Source: interface.go
+
+package server
```

Comment:
Yeah, I think we should consider this when we add an interface type in the `client` package. For now I'm fine generating this mock and using it as a mock of the client temporarily